### PR TITLE
feat: update auro-library to 3.0.1

### PR DESCRIPTION
**Pull request by DukeFerdinand**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Resolves:** #142 
## Summary:

This updates to the latest auro-library version and ensures the documentation paths are updated in `index.md` and `api.md` to match!

## Type of change:

Please delete options that are not relevant.

- [x] New capability
- [ ] Revision of an existing capability
- [x] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

## Summary by Sourcery

Update auro-library to version 3.0.1 and adjust documentation paths to match the new directory structure.

New Features:
- Update auro-library to version 3.0.1.

Documentation:
- Update documentation paths in index.md and api.md to reflect new directory structure.

### CODE REVIEW

**Review by chrisfalaska — COMMENTED** [review]
While running `npm run test` I received a warning:

```
auro-accordion/src/auro-accordion.js
  130:1  warning  The type 'TemplateResult' is undefined  jsdoc/no-undefined-types
```

**Review by chrisfalaska — DISMISSED** [review]
Approved with one comment.

File: docTemplates/README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ The README.md file is a compiled document. No edits should be made directly to t
 README.md is created by running `npm run build:docs`.
 
 This file is generated based on a template fetched from
-`https://raw.githubusercontent.com/AlaskaAirlines/WC-Generator/master/componentDocs/README.md`
```
Comment (chrisfalaska) [inline docTemplates/README.md:7]: `docTemplates` should be added to `gitignore`

**Review by chrisfalaska — CHANGES_REQUESTED** [review]

**Review by jason-capsule42 — APPROVED** [review]

**Review by chrisfalaska — APPROVED** [review]
Tested locally 🚀

**Pull request merged**